LinkedList/AddTwoLinkedList.py

```
@@ -0,0 +1,62 @@
+              self.tail=node 
+              return 
+          self.tail.next=node 
+          self.tail=node 
+              
+      def addTwoNumbers(self, l1:ListNode, l2:ListNode):
+          carry=0
+          l1=self.reverse(l1)
+          l2=self.reverse(l2)
+          result=None 
+          while l1 and l2:
+                sum=l1.val+l2.val+carry 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)